src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -85,8 +88,8 @@ csync_vio_handle_t *csync_vio_local_opendir(const char *name) {
 
   handle->firstFind = 1; // Set a flag that there first fileinfo is available.
 
-  handle->path = c_strdup(name);
-  c_free_locale_string(dirname);
+  dirname[std::wcslen(dirname)-1] = L'\0'; // remove the *
```

Comment:
Previously the path didn't have a trailing /, it seems

Reply:
correct, but it was not used so i allowed myself to change the behavior there.